addons/File_Sender/src/presenter.js

```diff
@@ -669,6 +673,12 @@ function AddonFile_Sender_create() {
         }catch(e){console.log(e)}
     };
 
+    presenter.isMauthor = function () {
```

Comment:
Nie podoba mi się ten warunek... ale mam wątpliwości czy tak może być. Gdyby był to dowolnie inny addon to nie byłoby akceptacji na to.
Mam wątpliwości dlatego, że na mAuthor bez zamokowania pewnych danych nie będzie możliwe przetestowanie tego. Ale jeżeli chciałoby się jednak przetestować z użyciem moków to doprowadzimy do innego zachowania na mA i mC, co nie jest akceptowalne.

Jeżeli w zadaniu uznają że ma to być poprawka pod mC to nie będę się czepiać tego sprawdzenia.